src/jit/codegencommon.cpp

```diff
@@ -8760,10 +8760,10 @@ void CodeGen::genFnProlog()
     genZeroInitFrame(untrLclHi, untrLclLo, initReg, &initRegZeroed);
 
 #if FEATURE_EH_FUNCLETS
-
+#ifndef _TARGET_X86_
     genSetPSPSym(initReg, &initRegZeroed);
```

Comment:
This `#ifdef` is unnecessary: genSetPSPSym() will bail out early because `lvaPSPSym == BAD_VAR_NUM` which is true because `ehNeedsPSPSym()` will return false (so `lvaMarkLocalVars()` won't create one).